app/src/main/res/layout/activity_sign_in.xml

```diff
@@ -1,35 +1,50 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.constraint.ConstraintLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    tools:context=".ui.signin.SignInActivity">
+    android:layout_width="match_parent" android:layout_height="match_parent"
+    android:background="@color/colorPrimary">
 
-    <EditText
-        android:id="@+id/editText"
+
+    <ImageView
+        android:id="@+id/sigin_image_logo"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:ems="10"
-        android:hint="@string/hint_email"
```

Comment:
Kindly add tools:text for the characters visibility
Go through the below link
https://developer.android.com/studio/write/tool-attributes.html

Reply:
file is deleted

Reply:
We can think of using in existing xml files